flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/sinks/BatchTableSink.java

```diff
@@ -31,5 +32,5 @@
 public interface BatchTableSink<T> extends TableSink<T> {
 
 	/** Emits the DataSet. */
-	void emitDataSet(DataSet<T> dataSet);
+	DataSink<?> emitDataSet(DataSet<T> dataSet);
```

Comment:
`consumeDataSet`?

Reply:
> `consumeDataSet`?

agree, although the change is a little larger. `consumeDataSet` aligns with `consumeDataStream`